02-express-tutorial/answers/01-http-basics.js

```diff
@@ -0,0 +1,26 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+  // console.log(req.method)
+  const url = req.url
+  // home page
+  if (url === '/') {
```

Comment:
I appreciate that your code is looking very clean and well-formatted. Whether you're using a tool or just giving more attention to detail, your work is looking more professional and high-quality; nice job!